package.json

```diff
@@ -2,13 +2,13 @@
     "name": "scala",
     "displayName": "Scala Syntax (official)",
     "description": "Official Scala Syntax",
-    "version": "0.3.4",
+    "version": "0.3.5",
```

Comment:
This repo has automatic CI releases on git tag push. I believe that package.json is updated from Travis along with release notes https://github.com/scala/vscode-scala-syntax/commit/f3d587d75ac8ca92c006ff72a76c8a4a0158ed76

Reply:
The hard work is writing release notes that explain the latest changes 😄 https://github.com/scala/vscode-scala-syntax/releases